src/behavior-considered-undefined.md

```diff
@@ -23,44 +23,56 @@ code.
 </div>
 
 * Data races.
-* Dereferencing a null or dangling raw pointer.
-* Unaligned pointer reading and writing outside of [`read_unaligned`]
```

Comment:
Can you explain removing this?

Reply:
It is redundant: unaligned references are UB when produced, unaligned pointers are still called out, the rest is `unsafe` lib functions and nothing language-inherent.

Reply:
Indeed it is; 👍